client/src/pages/home/components/events/constants.jsx

```diff
@@ -11,16 +10,8 @@ import summerHiring from 'assets/home-page/events/summer_hiring.jpeg';
 
 export const UPCOMINGEVENTS = {
 	title: 'Upcoming Events',
-	events: [
-		{
-			title: 'Summer Hiring',
-			desc: ['Now - June 6 11:59 PM'],
-			img: <img src={summerHiring} alt='Summer Hiring' />,
-			link: 'drive.google.com/file/d/1pznE1l3oLW-n5KpCiyQ6UWNShFJgJ-fQ/view?usp=drive_link',
-		},
-	],
+	events: '',
 	icon: <img src={upcomingEvent} alt='' className='ml-[25%] w-[80%] sm:ml-[35%] sm:w-full' />,
-	button1: 'Register',
```

Comment:
Keep this line. If the data (events) is empty, it won't show the first button anyway. This is a bad layout design indeed, so no need to worry :).